src/Editmode.jsx

```diff
@@ -1,23 +1,28 @@
 // @ts-check
 import React, { useEffect } from "react";
 import { EditmodeContext } from "./EditmodeContext";
+import { Platform } from 'react-native';
```

Comment:
We can't safely depend on `react-native` within a web environment because of how `react-native` publishes to NPM (under the expectation that it's bundled via `expo` or `metro` bundler)